Source/WebDriver/WebDriverService.h

```diff
@@ -43,13 +44,17 @@ class CommandResult;
 class Session;
 
 class WebDriverService final : public HTTPRequestHandler
+    , public WTF::RefCounted<WebDriverService>
```

Comment:
I'm not sure I understand why the WebDriverService needs to be ref-counted, IIUC it's only instantiated once in `main()` and its lifespan is bound to the program execution, right?

Reply:
Yes, its lifespan is bound to the `main()` method. But `WebDriverService` also implements a couple of interfaces that are held as weakptr (`HTTPRequestHandler` and `WebSocketMessageHandler`) in other classes.

Initially I tried making `WebDriverService` just a `WeakPtr`, but, IIRC, I had issues with the diamond inheritance through the two handler classes. So I ended up having to add some concrete ref/deref impl, which in this case comes from `RefCounted`. But I'll take another look at it.